includes/express-checkout/class-wc-payments-express-checkout-button-display-handler.php

```diff
@@ -97,7 +97,6 @@ public function init() {
 
 		if ( $is_woopay_enabled || $is_payment_request_enabled ) {
 			add_action( 'wc_ajax_wcpay_add_to_cart', [ $this->express_checkout_ajax_handler, 'ajax_add_to_cart' ] );
-			add_action( 'wc_ajax_wcpay_empty_cart', [ $this->express_checkout_ajax_handler, 'ajax_empty_cart' ] );
```

Comment:
I can't find the invocation of this anywhere, I assume this wasn't even used?

Reply:
Correct, I couldn't find its usage in the current codebase.